### ISSUE TITLE
Support stricter type for Deno 1.4

### ISSUE THREAD

**Issue by bebraw** [ryo-ma/deno-websocket#10]
It looks like Deno 1.4 is stricter with typing.

The fix is to change `WebSocket` to be exported as a type at https://github.com/ryo-ma/deno-websocket/blob/master/deps.ts .

```ts
export type {
  WebSocket,
} from "https://deno.land/std@0.65.0/ws/mod.ts";
```

More specifically, it looks like the issue originates from https://deno.land/posts/v1.4#stricter-type-checks-in-code--unstablecode (`--unstable` flag).

**Comment by ryo-ma** [comment]
I will consider supporting Deno 1.4.
But I think deno-websocket is losing its existence value due to WebSocket API.
https://deno.land/posts/v1.4#websocket-api

Do you need this library?

**Comment by bebraw** [comment]
If I'm interpreting right, that's only the **client** portion. Please correct me if I'm wrong. 😄

**Comment by ryo-ma** [comment]
I see! It may be exactly what you say. 
I will check in detail later.

**Comment by bebraw** [comment]
Cool, it looks like the fix is simple. To be future-compatible, all we have to do is to change the export.

**Comment by ryo-ma** [comment]
Released v0.0.4. Please confirm it.
Then I will update std lib later.

**Comment by bebraw** [comment]
It looks like the current fix fails with

```
TS1362 [ERROR]: 'WebSocketServer' cannot be used as a value because it was exported using 'export type'.
  const wss = new WebSocketServer(8080);
```

In my local branch, changing **deps.ts** to

```
export { EventEmitter } from "https://deno.land/std@0.65.0/node/events.ts";
export { Server, serve } from "https://deno.land/std@0.65.0/http/server.ts";
export {
  acceptWebSocket,
  isWebSocketCloseEvent,
  isWebSocketPingEvent,
  isWebSocketPongEvent,
  connectWebSocket,
} from "https://deno.land/std@0.65.0/ws/mod.ts";
export type {
  WebSocket,
} from "https://deno.land/std@0.65.0/ws/mod.ts";
```

was enough given `WebSocket` is the only type we expose as far as I understand.

**Comment by ryo-ma** [comment]
I made a mistake.
I have fixed it. And I have released v0.0.5.

**Comment by bebraw** [comment]
Great, looks like that one works well. Thanks for the fix. 👍
